src/uiSelectController.js

```diff
@@ -257,8 +254,6 @@ uis.controller('uiSelectCtrl',
       if (ctrl.dropdownPosition === 'auto' || ctrl.dropdownPosition === 'up'){
         $scope.calculateDropdownPos();
       }
-
-      $scope.$broadcast('uis:refresh');
```

Comment:
No harm is keeping it there now, someone might use it already and we should avoid breaking their use case 😬